docs/rst/recording/tutorials/basic_example.rst

```diff
@@ -2,9 +2,263 @@
+    :language: C++
+    :lines: 45-53
+
+Inside the ``TypeLookupServiceSubscriber`` constructor are defined the DomainParticipantQos.
+As the publisher act as a client of types, set the QoS in order to receive this information.
+Set ``use_client`` to ``true`` and ``use_server`` to ``false``.
+
+.. literalinclude:: ../../../../resources/dds/TypeLookupService/TypeLookupServiceSubscriber.cpp
+    :language: C++
+    :lines: 57-61
+
+Then is initialized the Subscriber.
+
+Inside ``on_data_available()`` it is created the ``DynamicData_ptr`` where the samples received will be read.
+
+As in the publisher, it is used ``DynamicDataFactory`` for the creation of the data that corresponds to our data type.
+
+.. literalinclude:: ../../../../resources/dds/TypeLookupService/TypeLookupServiceSubscriber.cpp
+    :language: C++
+    :lines: 143-170
+
+The function ``on_type_information_received()`` detect the topics received and discover it if they have the same as the topic that we are waiting for.
+Discover the topic means set the topic as discover and register it using the function ``register_remote_type_callback_()`` explained below.
+Once the topic has been discovered and registered, it is created a DataReader on this topic.
+
+.. literalinclude:: ../../../../resources/dds/TypeLookupService/TypeLookupServiceSubscriber.cpp
+    :language: C++
+    :lines: 172-212
+
+Finally, the function ``register_remote_type_callback_()`` is in charge of register the topic received.
+First of all, it creates a ``TypeSupport`` with the corresponding type and register it into the participant.
+Then, it creates the DDS Topic with the topic name set in the creation of the Subscriber and the topic type previously register.
+Finally, it creates the DataReader of that topic.
+
+.. literalinclude:: ../../../../resources/dds/TypeLookupService/TypeLookupServiceSubscriber.cpp
+    :language: C++
+    :lines: 278-304
+
+***********************
+Running the application
+***********************
+
+Recording samples with DDS Recorder
+===================================
+
+Open two terminals:
+
+In the first terminal, run:
+
+.. code-block:: bash
+
+    source install/setup.bash
+    cd DDS-Recorder/build/TypeLookupService
+    ./TypeLookupService
+
+In the second terminal, run the ddsrecorder:
+
+.. code-block:: bash
+
+    source install/setup.bash
+    ddsrecorder
+
+Publisher <-> Subscriber
+========================
+
+Open two terminals:
+
+In the first terminal, run the DDS Publisher:
+
+.. code-block:: bash
+
+    source install/setup.bash
+    cd DDS-Recorder/build/TypeLookupService
+    ./TypeLookupService --entity publisher
+
+In the second terminal, run the DDS Subscriber:
+
+.. code-block:: bash
+
+    source install/setup.bash
+    cd DDS-Recorder/build/TypeLookupService
+    ./TypeLookupService --entity subscriber
+
+.. figure:: /resources/tutorials/basic_publisher_subscriber.png
```

Comment:
```suggestion
At this point, we observe that the data published reach the subscriber and it can access to the content of the sample received. 

.. figure:: /resources/tutorials/basic_publisher_subscriber.png
```